src/documentation/pages/components/tooltip/examples/_light_tooltip.vue

```diff
@@ -0,0 +1,66 @@
+<template lang="pug">
+  <ComponentExample title="Light" id="light" padding="-p--0" :tabs="exampleTabs">
```

Comment:
Light is only for Lumen and CenturLink
```suggestion
  <ComponentExample title="Light" id="light-lumen-centurylink" padding="-p--0" :tabs="exampleTabs">
```